libc/src/stdlib/CMakeLists.txt

```diff
@@ -432,6 +433,28 @@ add_entrypoint_object(
     libc.src.__support.threads.mutex
 )
 
+add_entrypoint_object(
+  atexit
+  SRCS
+    atexit.cpp
+  HDRS
+    atexit.h
+  CXX_STANDARD
```

Comment:
this doesn't have anything constinit in it anymore, so it doesn't need this `CXX_STANDARD` annotation.